docs/debian.md

```diff
@@ -1,7 +1,3 @@
 # Debian repository
 
-To use this repository with your Debian machine, create a file `solaar.list` in
-`/etc/apt/sources.list.d/`, with the following contents:
-
-	deb https://pwr.github.io/Solaar/packages/ ./
-	deb-src https://pwr.github.io/Solaar/packages/ ./
+Solaar is now part of the [official debian repository](https://packages.debian.org/en/stretch/solaar-gnome3), to install it on your debian machine, use the following command: `sudo apt install solaar-gnome3`
```

Comment:
You can also drop /en/ from the URL. Even better, drop /stretch/ as well so we do not have to update this for every release:
https://packages.debian.org/solaar-gnome3

Reply:
Thank you for the information! I updated it.